build.fsx

```diff
@@ -9,23 +9,11 @@ open System
 let release = LoadReleaseNotes "RELEASE_NOTES.md"
 let srcGlob = "*.csproj"
 
-
 Target "Clean" (fun _ ->
     [ "obj" ;"dist"]
     |> CleanDirs
     )
 
-Target "DotnetRestore" (fun _ ->
```

Comment:
Why remove this target ? It's the one make restoring the deps for the projects

Reply:
Didn't think it was necessary for the top level project anymore.